src/index-metadbs/matcloud/v1/links.json

```diff
@@ -0,0 +1,34 @@
+{
+   "meta" : {
+       "api_version" : "1.0.0",
+       "query" : {
+           "representation" : "/links"
+       },
+       "more_data_available" : false,
+       "schema" : "https://schemas.optimade.org/openapi/v1.0/optimade_index.json"
+   },
+   "data" : [
+      {
+         "id" : "matcloud",
+         "type" : "links",
+         "attributes" : {
+            "name": "An integrated high-throughput computational materials platform",
+            "description": "MatCloud is a high-throughput computing platform integrating data, simulation and supercomputing.",
+            "base_url": "http://207.246.86.9:5000/optimade",
```

Comment:
You don't have a named domain to put in here instead? As otherwise you will have to change this again later once you get that.

Furthermore, The OpenAPI schema shown there does not reflect what would be expected from the named endpoints. See the specification [here](https://github.com/Materials-Consortia/OPTIMADE/blob/develop/optimade.rst) to understand better how to setup your OPTIMADE implementation (e.g., endpoint names should consist of lowercase alphanumerics and usually be in the plural form, e.g., `/structures` instead of `/Structure`).

_Originally posted by @CasperWA in https://github.com/Materials-Consortia/providers/pull/61#discussion_r584574001_

Reply:
Our website is still under construction, and only some of the functions of the optimade specification have been implemented. And for these irregularities, I will carefully check and modify. 

Later, the domain name will be unified under MatCloud (http://matcloud.cnic.cn/).
Thank you for your suggestion.

_Originally posted by @jpdong00 in https://github.com/Materials-Consortia/providers/pull/61#discussion_r585183842_

Reply:
Just to add my two cents here, I would suggest setting your `base_url` here to `null` for now, until you have finished your implementation. This will reserve the appropriate prefix and signal the intent of matcloud to implement OPTIMADE, but it will prevent your database being used by all OPTIMADE clients/validators until it is finished.

You may be interested in using the validator we have written at (https://github.com/Materials-Consortia/optimade-python-tools) to help finish off your implementation. It is also available as GitHub action (https://github.com/Materials-Consortia/optimade-validator-action) and as a no-install binder repository (https://mybinder.org/v2/gh/ml-evs/optimade-validator-binder/master?filepath=optimade_validator.ipynb) where you can test your current URL.